include/RAJA/util/CombiningAdapter.hpp

```diff
@@ -0,0 +1,264 @@
+/*!
+ ******************************************************************************
+ *
+ * \file
+ *
+ * \brief   Header file for RAJA CombingAdapter.
+ *
+ ******************************************************************************
+ */
+
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
```

Comment:
Make sure you update the copyright years in the new files you added to match the other source files.

Reply:
fixed